src/server/node/bot/src/services/rt-services.ts

```diff
@@ -0,0 +1,22 @@
+import { connectToGateway } from '@adaptive/hydra-platform'
+import { noop } from 'rxjs'
+import logger from '../logger'
+
+const { Crypto } = require('@peculiar/webcrypto')
+
+// Global packages used by hydra-platform that aren't available in node by default
+Object.assign(global, {
+  WebSocket: require('ws'),
+  crypto: new Crypto()
```

Comment:
Polyfill is needed as `crypto` is used in `hydra-platform` but isn't defined in node. 
Node's version of `crypto` doesn't have the same api as the browser, so using this polyfill instead.
An improvement could be to inject the `crypto` provider into hydra through `connectToGateway`, rather than assigning to global.